python-librarian-simplified/python-librarian-simplified.mk

```diff
@@ -0,0 +1,17 @@
+################################################################################
+#
+# python-librarian-simplified
+#
+################################################################################
+
+PYTHON_LIBRARIAN_SIMPLIFIED_VERSION = 369bd53502681f5d410e2bfb2454c0a754e774ad
+PYTHON_LIBRARIAN_SIMPLIFIED_SITE = $(call github,Outernet-Project,librarian-simplified,$(PYTHON_LIBRARIAN_SIMPLIFIED_VERSION))
+PYTHON_LIBRARIAN_SIMPLIFIED_LICENSE = GPLv3+
+PYTHON_LIBRARIAN_SIMPLIFIED_LICENSE_FILES = COPYING
+PYTHON_LIBRARIAN_SIMPLIFIED_SETUP_TYPE = setuptools
+
+ifeq ($(BR2_PACKAGE_PYTHON_LIBRARIAN_SIMPLIFIED),y)
```

Comment:
Probably shouldn't be added by default. This is something overlays should take care of.